Tests/Opc.Ua.Gds.Tests/Opc.Ua.GlobalDiscoveryTestServer.Config.xml

```diff
@@ -149,6 +149,7 @@
         </CertificateGroups>
         <KnownHostNames/>
         <DatabaseStorePath>%LocalApplicationData%/OPC/GDS/gdsdb.json</DatabaseStorePath>
+        <UsersDatabaseStorePath>%LocalApplicationData%/OPC/GDS/gdsusersdb.json</UsersDatabaseStorePath>
```

Comment:
if the Configuration.xsd exists somewhere the field must be added.

Reply:
I added the property in [GlobalDiscoveryServerConfiguration.cs](https://github.com/OPCFoundation/UA-.NETStandard/pull/2402/files/f0b97302d1b61b77837c5d3457d024d099b6c89e#diff-e858f07238d1732fc88edd9a37d88bb0beb090e1f9ab1d2881d78dcc8979eb90) 
did I miss something?

Reply:
the good news, looks like there is no xsd file checked in for the config...